### PR TITLE
Update manage-alerts-windows-defender-advanced-threat-protection.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2287
Added info to alert classification.

### CODE REVIEW

**Comment by illfated** [comment]
Looks good to me (tm). :)

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -104,8 +104,7 @@ Alternatively, the team leader might assign the alert to the **Resolved** queue
 
 
 ## Alert classification
-You can choose not to set a classification, or  specify if an alert is a true alert or a false alert.
-
+You can choose not to set a classification, or  specify if an alert is a true alert or a false alert. It's important to provide the classification of true positive/false positive. This classification is used to monitor alert quality to help tune alerts to be more accurate using this feedback. The "determination" field defines additional fidelity for a "true positive" classification. The determination contains values for "security testing" to address alerts triggered by intended suspect activity such as pen-testing, which are true positives from a detection perspective, but it's intended.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md]:
```suggestion
You can choose not to set a classification, or specify whether an alert is a true alert or a false alert. It's important to provide the classification of true positive/false positive. This classification is used to monitor alert quality to help tune alerts to be more accurate by using this feedback. The "determination" field defines additional fidelity for a "true positive" classification. The determination contains values for "security testing" to address alerts triggered by intended suspect activity such as pen-testing, which are true positives from a detection perspective, but it's intended.
```

File: windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md [hunk 2]
```diff
@@ -104,8 +104,7 @@ Alternatively, the team leader might assign the alert to the **Resolved** queue
 
 
 ## Alert classification
-You can choose not to set a classification, or  specify if an alert is a true alert or a false alert.
-
+You can choose not to set a classification, or specify whether an alert is a true alert or a false alert. It's important to provide the classification of true positive/false positive. This classification is used to monitor alert quality to help tune alerts to be more accurate by using this feedback. The "determination" field defines additional fidelity for a "true positive" classification. The determination contains values for "security testing" to address alerts triggered by intended suspect activity such as pen-testing, which are true positives from a detection perspective, but it's intended.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md]:
```suggestion
You can choose not to set a classification, or specify whether an alert is a true alert or a false alert. It's important to provide the classification of true positive/false positive. This classification is used to monitor alert quality, and make alerts more accurate. The "determination" field defines additional fidelity for a "true positive" classification. 
```

**Comment by lindspea** [comment]
Ok, do I simplify or does Johan?

**Comment by e0i** [comment]
@mjcaparas Could kindly review and merge this PR based on issue #2287? Thanks.

**Pull request merged**